plugins/python-build/share/python-build/pyston-0.5.1

```diff
@@ -0,0 +1,19 @@
+case "$(pypy_architecture 2>/dev/null || true)" in
```

Comment:
Done in https://github.com/yyuu/pyenv/pull/718/commits/d24ce2494bc4d2890368b9f15c9d5531e61dcb31 and https://github.com/yyuu/pyenv/pull/718/commits/4e4bd420202bcc16da696002171fafd007dae28f